tsconfig.node.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
I was wondering if we need to run a custom express server in the first place — we could probably just use the react-router server, and simplify the tsconfig setup, and get rid of managing server.js ourselves?